airsonic-main/src/test/java/org/airsonic/player/api/jukebox/AbstractAirsonicRestApiJukeboxIntTest.java

```diff
@@ -144,6 +135,12 @@ public void setup() throws Exception {
         assertThat(testJukeboxPlayer.getPlayQueue().size()).isEqualTo(2);
     }
 
+    @After
+    public void cleanDataBase() {
+        daoHelper.getJdbcTemplate().execute("DROP SCHEMA PUBLIC CASCADE");
+        dataBasePopulated = false;
+    }
```

Comment:
Any idea why this is part of the PR?

It'll slow down the tests... Was it polluting the db for the next test? Aren't the tests just start and stop jukebox?

Reply:
I am not really satisfied with this, but this was the only way I could find to actually run these tests without running into issues of the db being polluted. I did not have too much time to investigate, but do you see better ways?

Reply:
Not sure why your db is getting polluted between the tests, but I don't see anything in the tests that is writing to the db. They all seem to be read operations. They just start and stop jukeboxes for two different jukeboxes. I don't think this should be part of the PR.

Reply:
There is more info in the original PR in https://github.com/airsonic/airsonic/pull/876. I'll probably try to get it working at some point, but this'll take more digging.

Reply:
hmm, that doesn't look like a test pollution. That looks like just the db shutdown for some reason. Don't know between which tests (in the same suite or the different suite)

Try removing line 41 so it stops trying to dirty the context cache (so it doesn't shut down) and see if that works?